### PR TITLE
Add tooltips to deploy dialog

### LINKED ISSUE #1124: Tooltips for everything in the Deployment Dialog
image

### PR DESCRIPTION
**Pull request by chanseokoh**
Fixes #1124.

Most of the tooltip text are taken from `gcloud app deploy --help`. I verified that every tooltip is working.

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java [hunk 1]
```diff
@@ -270,12 +270,14 @@ public Credential getSelectedCredential() {
   private void createCredentialSection(IGoogleLoginService loginService) {
     Composite accountComposite = new Composite(this, SWT.NONE);
 
-    new Label(accountComposite, SWT.LEFT).setText(
-        Messages.getString("deploy.preferences.dialog.label.selectAccount"));
+    Label accountLabel = new Label(accountComposite, SWT.LEFT);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java]: SWT.LEFT --> SWT.LEAD for i18n

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java [hunk 2]
```diff
@@ -284,10 +286,12 @@ private void createProjectIdSection() {
 
     projectIdLabel = new Label(projectIdComposite, SWT.LEFT);
     projectIdLabel.setText(Messages.getString("project.id"));
+    projectIdLabel.setToolTipText(Messages.getString("tooltip.project.id"));
     GridData layoutData = GridDataFactory.swtDefaults().create();
     projectIdLabel.setLayoutData(layoutData);
-    
+
     projectId = new Text(projectIdComposite, SWT.LEFT | SWT.SINGLE | SWT.BORDER);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java]: SWT.LEFT --> SWT.LEAD for i18n

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java [hunk 3]
```diff
@@ -284,10 +286,12 @@ private void createProjectIdSection() {
 
     projectIdLabel = new Label(projectIdComposite, SWT.LEFT);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java]: SWT.LEFT --> SWT.LEAD for i18n

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties [hunk 4]
```diff
@@ -30,6 +29,15 @@ appengine.java.component.missing=Could not deploy project because the App Engine
  at the command line and try again.
 cloudsdk.not.configured.detail=Could not deploy project because the Cloud SDK was not found.\
  Check Preferences > Google Cloud Tools > SDK location and try again.
+tooltip.account=Google Account used to deploy the app to the Google Cloud Platform project.
+tooltip.project.id=The Google Cloud Platform project where the app will be deployed to.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties]: delete "to"

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties [hunk 5]
```diff
@@ -30,6 +29,15 @@ appengine.java.component.missing=Could not deploy project because the App Engine
  at the command line and try again.
 cloudsdk.not.configured.detail=Could not deploy project because the Cloud SDK was not found.\
  Check Preferences > Google Cloud Tools > SDK location and try again.
+tooltip.account=Google Account used to deploy the app to the Google Cloud Platform project.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties]: delete "project"

File: plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties [hunk 6]
```diff
@@ -30,6 +29,15 @@ appengine.java.component.missing=Could not deploy project because the App Engine
...
+tooltip.project.id=The Google Cloud Platform project where the app will be deployed to.
+tooltip.version=The version of the app that will be created or replaced by this\
+ deployment. If you do not specify a version, one will be generated for you.
+tooltip.manual.promote.link=To manually promote a version use the Google Cloud Console: {0}
+tooltip.stop.previous.version=If checked, stops the previously running version when deploying a new\
+ version that receives all traffic.
+tooltip.staging.bucket=The Google Cloud Storage bucket used to stage files associated with the\
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties]: associated with --> for

**Comment by chanseokoh** [comment]
Review comments addressed.

**Pull request merged**